src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -84,6 +98,38 @@ class FileColumnProperties {
     this.setDistinctValues(stats.getDistinctValues());
   }
 
```

Comment:
I see that these two methods are equivalent, trying to understand why we couldn't use RowBufferStats and had to rely on Statistics<?> instead?

Reply:
Switch back to `RowBufferStats`, remove this method.